### ISSUE TITLE
About defineStar

### ISSUE THREAD

**Issue by hh-in-zhuzhou** [cosinekitty/astronomy#283]
First thank u for your work. I use this library to write a planet image which was used to study culture calendar's secret, its data computed by this library is very accuracy。
second，I want to find Right ascension of planet in thousands of years, and in this years, I want to get some fixed star's Right ascension, you know the fixed star's Right ascension is changed, I found the defineStar function，but it only can support 8 stars，but I need to define 28 starts，I had read the code, but I cannot confirm if I can simply modify the source code to support 28 stars，please give me some suggestions!
third, I have a question, can u help me?  I want to convert a right ascension to ecliptic latitude angle, no need of latitude。there are many information in web about Coordinate system conversion， but it's not I need.  I find that it has a simple relation between right ascension and ecliptic latitude angle, but I don't know exactly what is. thank u.

**Comment by cosinekitty** [comment]
You can define up to 8 stars simultaneously, but you can also use any number of stars you want if you keep calling `DefineStar` each time before you use the star. It's a little less convenient, but `DefineStar` is a very efficient function so it will not slow down your code very much.

When you call `DefineStar`, you pass in the declination (DEC) and right ascension (RA) using the Earth's mean equator of 2000, called EQJ in Astronomy Engine. Then you can call the [Equator](https://github.com/cosinekitty/astronomy/tree/master/source/js#equatorbody-date-observer-ofdate-aberration--equatorialcoordinates) function to find the equatorial coordinates of the star (or any planet) at any date in the past or future. Pass `ofdate = true` to obtain coordinates corrected for the Earth's equator as it exists at the specified time.

To get ecliptic coordinates, call the [GeoVector](https://github.com/cosinekitty/astronomy/tree/master/source/js#GeoVector) function to obtain the position of a star or planet as a vector. Then pass that vector to the [Ecliptic](https://github.com/cosinekitty/astronomy/tree/master/source/js#eclipticeqj--eclipticcoordinates) function to convert to ecliptic angles, including the ecliptic latitude.

By the way, it is not possible to calculate ecliptic latitude if all you know is equatorial right ascension. You need to know both the right ascension and declination to calculate ecliptic latitude. (The same is true for ecliptic longitude.)

I hope this helps!

**Comment by hh-in-zhuzhou** [comment]
Thank u very much! 
Yestoday I had tried to modify source code to support define 28 stars simultaneously, luckily it's very simple, and I succeed to done the task, the result satisfied me, it's right enough. 
In China, we use 28 traditional Constellations between Ecliptic and equator to locate all the star inluding planet（each Constellations select one location star, use Right ascension degree, the Constellations distributed from 0 to 360 degree），these Constellations's location star's Right ascension changes in thousand years, so if I draw planet at special year，must correct the Right ascension of location star at that year.  then I need to define 28 stars simultaneously.

secondly, thanks for your suggestion about calculating ecliptic latitude,  why I have this question, I Carefully observed a sphere，the longitude of equator means a vertical cross section which through the Spheroid，it also cross the cross section of Ecliptic，the intersection point at sphere surface is the longitude of ecliptic, the relationship between them is obvious, but I cannot make the result of relationship, it looks be a Geometric puzzle......
